packages/amplify-e2e-tests/src/amplify-app-helpers/amplify-app-validation.ts

```diff
@@ -12,7 +12,6 @@ function validateProject(projRoot: string, platform: string) {
       expect(fs.existsSync(path.join(projRoot, 'app', 'src', 'main', 'res', 'raw', 'amplifyconfiguration.json'))).toBe(true);
       break;
     case 'ios':
-      expect(fs.existsSync(path.join(projRoot, 'amplifytools.xcconfig'))).toBe(true);
```

Comment:
AmplifyTools script has been deprecated in [Amplify iOS 1.6.1](https://github.com/aws-amplify/amplify-ios/releases/tag/v1.6.1), and `amplifytools.xcconfig` isn't supported anymore.